app/views/partials/_navbar.html.erb

```diff
@@ -33,4 +33,5 @@
       </form>
     </div>
   </div>
-</nav>
\ No newline at end of file
+</nav>
+<br>
```

Comment:
here also indentation fix, and can u also add extra line to the bottom of file

Reply:
sure